patches/chrome-android-java-src-org-chromium-chrome-browser-notifications-channels-ChannelsUpdater.java.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelsUpdater.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelsUpdater.java
+index d49ba4549b2a4de6e3d3503d3bb664d9b686b8c5..666639f90fbd1d3d00af6778c195d1e9c6b006a5 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelsUpdater.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelsUpdater.java
+@@ -10,6 +10,7 @@ import android.os.Build;
+ import androidx.annotation.VisibleForTesting;
+ 
+ import org.chromium.base.ContextUtils;
++import org.chromium.chrome.browser.notifications.BraveNotificationManagerProxyImpl;
+ import org.chromium.chrome.browser.notifications.NotificationManagerProxyImpl;
+ 
+ /**
+@@ -36,7 +37,7 @@ public class ChannelsUpdater {
+         public static final ChannelsUpdater INSTANCE = Build.VERSION.SDK_INT < Build.VERSION_CODES.O
+                 ? new ChannelsUpdater(false /* isAtLeastO */, null, null, -1)
+                 : new ChannelsUpdater(true /* isAtLeastO */, ContextUtils.getAppSharedPreferences(),
+-                          new ChannelsInitializer(new NotificationManagerProxyImpl(
++                          new ChannelsInitializer(new BraveNotificationManagerProxyImpl(
```

Comment:
we can use `new org.chromium.chrome.browser.notifications.BraveNotificationManagerProxyImpl(...)` to avoid extra patching in imports

Reply:
Good point, will fix.

Reply:
Fixed.